compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3079,6 +3079,67 @@ object Parsers {
 
  /* -------- PARAMETERS ------------------------------------------- */
 
+    /** DefParamClause ::= DefTypeParamClause 
+     *                   | DefTermParamClause 
+     *                   | UsingParamClause
+     */
+    def typeOrTermParamClause(
+                    ownerKind: ParamOwner,
+                    nparams: Int,                    // number of parameters preceding this clause
+                    ofClass: Boolean = false,        // owner is a class
+                    ofCaseClass: Boolean = false,    // owner is a case class
+                    prefix: Boolean = false,         // clause precedes name of an extension method
+                    givenOnly: Boolean = false,      // only given parameters allowed
+                    firstClause: Boolean = false     // clause is the first in regular list of clauses
+                   ): List[TypeDef] | List[ValDef] =
+      if (in.token == LPAREN)
+        termParamClause(nparams, ofClass, ofCaseClass, prefix, givenOnly, firstClause)
+      else if (in.token == LBRACKET)
+        typeParamClause(ownerKind)
+      else
+        Nil
+      
+    end typeOrTermParamClause
+    
+    /** DefParamClauses       ::= DefParamClause { DefParamClause }  -- and two DefTypeParamClause cannot be adjacent
+     */
+    def typeOrTermParamClauses(
+                    ownerKind: ParamOwner,
+                    ofClass: Boolean = false,
+                    ofCaseClass: Boolean = false,
+                    givenOnly: Boolean = false,
```

Comment:
There is only one call to this method and it doesn't specify these arguments explicitly.

Reply:
I left them to have the same signature as the other, should I simplify it instead ?